packages/core/src/node/initRsbuild.ts

```diff
@@ -83,11 +87,23 @@ async function createInternalBuildConfig(
     node: ['node >= 14'],
   };
 
+  const sassPluginIncluded =
+    config.builderPlugins?.some(plugin => plugin.name === 'rsbuild:sass') ||
+    config.builderConfig?.plugins?.some(
+      plugin => plugin && (plugin as RsbuildPlugin).name === 'rsbuild:sass',
+    );
+
+  const lessPluginIncluded =
+    config.builderPlugins?.some(plugin => plugin.name === 'rsbuild:less') ||
+    config.builderConfig?.plugins?.some(
+      plugin => plugin && (plugin as RsbuildPlugin).name === 'rsbuild:less',
+    );
+
   return {
     plugins: [
       pluginReact(),
```

Comment:
This can be applied to pluginReact too, which can allow users to custom pluginReact options.

Reply:
@Timeless0911 cc